### PR TITLE
Tstanwick ui editing pull

### PR DESCRIPTION
**Pull request by tstanwick**
Testing

### CODE REVIEW

**Review by vanderwalutc — APPROVED** [review]
So welcoming. Nice.

**Review by vanderwalutc — APPROVED** [review]
Nice

File: _posts/0000-01-10-tstanwick.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "I always preferred the joker over the riddler!"
```
Comment (vila89) [inline _posts/0000-01-10-tstanwick.md:3]: 👍

**Review by vila89 — APPROVED** [review]
Awesome! LGTM!

**Pull request merged**